package.json

```diff
@@ -164,7 +167,8 @@
     "webpack": "4.19.0",
     "webpack-bundle-analyzer": "2.13.1",
     "websocket": "1.0.28",
-    "window-size": "1.1.1"
+    "window-size": "1.1.1",
+    "ws": "6.1.2"
```

Comment:
Do we need `ws` since we already have `websocket` as a package?

Reply:
Good question. I couldn't find a component in `websocket` that provides a writable stream (implementing a parser) equivalent to `ws.Receiver`.